app/views/admin/auctions/_work_in_progress.html.erb

```diff
@@ -0,0 +1,4 @@
+<%= link_to I18n.t('statuses.admin_auction_status_presenter.work_in_progress.actions.mark_delivered'),
+    admin_auction_path(status.auction, auction: { delivery_status: 'pending_acceptance' }),
```

Comment:
I like that you are re-using the default controller. I feel like we should also do this for some of the other update links that are updating a single attr (right now we have custom controllers for some of those)